woocommerce/woocommerce
# Update Marketplace search component

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:


We're updating the marketplace search placeholder text to align with what we use on WCCOM, and also replacing our custom component with the `<SearchControl />` component included in WordPress.



Closes https://github.com/Automattic/woocommerce.com/issues/21595



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Check out this branch
2. Go to `/plugins/woocommerce` and run `pnpm --filter='@woocommerce/plugin-woocommerce' build`
3. Visit the [in-app marketplace](http://localhost:8888/wp-admin/admin.php?page=wc-admin&path=%2Fextensions).
4. Confirm the Search field top-right displays "Search Marketplace" as placeholder text
5. Confirm the Search field matches the designs (Figma link in linked issue)
6. Try searching for a plugin. Confirm you're able to trigger a search request and receive results
7. Confirm you can search using keyboard navigation only, and that VoiceOver announcements work as expected
8. Also check the size and positioning of the Search on mobile screen sizes

![Extensions_‹_WooCommerce_‹_WooCommerce_Core_E2E_Test_Suite](https://github.com/user-attachments/assets/d106cbf3-ca4f-43d4-9d47-9fe544a3afcd)
![Extensions_‹_WooCommerce_‹_WooCommerce_Core_E2E_Test_Suite](https://github.com/user-attachments/assets/c36853d6-f966-4af5-9392-5d691d5380a8)
![Extensions_‹_WooCommerce_‹_WooCommerce_Core_E2E_Test_Suite](https://github.com/user-attachments/assets/0a9a3753-a698-43db-bce0-9d005c9e7ffd)




### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 
Replace marketplace search component with SearchControl from @wordpress/components
</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
